.github/actions/labeler/src/labeler.ts

```diff
@@ -0,0 +1,122 @@
+import core from '@actions/core';
+import github from '@actions/github';
+
+type ClientType = ReturnType<typeof github.getOctokit>;
+
+interface PrFile {
+  sha: string;
+  filename: string;
+  // exists if status is 'renamed'
+  previous_filename?: string;
+  status: 'added' | 'renamed' | 'modified' | 'removed';
+  additions: number;
+  deletions: number;
+  changes: number;
+  blob_url: string;
+  raw_url: string;
+  contents_url: string;
+  patch: string;
+}
+
+export async function run() {
+  try {
+    main();
+  } catch (err) {
+    core.error(err as Error);
+    core.setFailed((err as Error).message);
+  }
+}
+export async function main() {
+  const token = core.getInput('token', { required: true });
+
+  const prNumber = github.context.payload.pull_request?.number;
+  if (!prNumber) {
+    console.log('Could not determine PR number, skipping');
+    return;
+  }
+
+  const client: ClientType = github.getOctokit(token);
+  const changedFiles = await getChangedFiles(client, prNumber);
+
+  const labels = new Set<string>();
+  const translatedPagePattern = /pages\.[a-z_]+/i;
+  const pagePattern = /pages(?:\.[a-z_]+)?/i;
+  const documentationPattern = /[a-z\-]+\.md/i;
+  const toolingPattern = /.*\.(py|[jt]s|sh)/i;
+  for (const file of changedFiles) {
+    if (pagePattern.test(file.filename) || (file.previous_filename && pagePattern.test(file.previous_filename))) {
+      if (translatedPagePattern.test(file.filename) || (file.previous_filename && translatedPagePattern.test(file.previous_filename))) {
+        labels.add('translation');
+      }
+      if (file.status === 'added') {
+        labels.add('new command');
```

Comment:
So a translation gets the labels `translation` and `new command`? "new command" should only be applied to new English pages.
Would this fix it?
```suggestion
      } else if (file.status === 'added') {
        labels.add('new command');
```

Reply:
What should happen if they add a new page along with a translated version of it? It should have both tags there, right?

Reply:
In this case, yes. Didn't think of that.